### PR TITLE
S2U-21 Tests & Quizzes: Several fixes for SEB

### PR DESCRIPTION
**Pull request by stetsche**
Fixes:

- [S2U-21-1](https://sakaiproject.atlassian.net/browse/S2U-21?focusedCommentId=210536)
- [S2U-21-4](https://sakaiproject.atlassian.net/browse/S2U-21?focusedCommentId=210536)
- [S2U-21-5](https://sakaiproject.atlassian.net/browse/S2U-21?focusedCommentId=210536)
- [S2U-21-7](https://sakaiproject.atlassian.net/browse/S2U-21?focusedCommentId=210536)
- [S2U-21-8](https://sakaiproject.atlassian.net/browse/S2U-21?focusedCommentId=210536)
- [S2U-21-9](https://sakaiproject.atlassian.net/browse/S2U-21?focusedCommentId=210536)

https://sakaiproject.atlassian.net/browse/S2U-21

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ConfirmPublishAssessmentListener.java [hunk 1]
```diff
@@ -91,6 +91,7 @@ public void processAction(ActionEvent ae) throws AbortProcessingException {
     SaveAssessmentSettings s = new SaveAssessmentSettings();
     AssessmentService assessmentService = new AssessmentService();
     AssessmentFacade assessment = assessmentService.getAssessment(assessmentId);
+  	boolean isFromAssessmentSettings = Boolean.TRUE.toString().equals(ContextUtil.lookupParam("fromAssessmentSettings"));
```
Comment (jesusmmp) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ConfirmPublishAssessmentListener.java:94]: Spacing

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties [hunk 2]
```diff
@@ -437,6 +437,8 @@ seb_exam_keys_upload_info=Per a configuracions pujades amb fitxer les claus de l
 Descarrega la configuraci\u00f3 (alterada) de nou i insereix les claus de l\u2019examen presents en el fitxer.
 seb_exam_keys_upload_published_info=Per favor, insereix les claus de l\u2019examen presents en el fitxer de configuraci\u00f3 que pots descarregar dalt. \
 La configuraci\u00f3 s\u2019ha alterat quan l\u2019examen es va publicar.
+seb_exam_keys_recommended=It is recommended to add exam keys for this type of configuration, because the Safe Exam Browser Configuration can \
+not be validated automatically
```
Comment (stetsche) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties:441]: @bgarciaentornos Could you help me with this translation please?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_es.properties [hunk 3]
```diff
@@ -444,6 +444,8 @@ seb_exam_keys_upload_info=Para configuraciones subidas mediante fichero las clav
 Vas a tener que descargar la configuraci\u00f3n (alterada) de nuevo e insertar las claves del examen presentes en ese fichero.
 seb_exam_keys_upload_published_info=Por favor, inserta las claves del examen presentes en el fichero de configuraci\u00f3n que puedes descargar de arriba. \
 La configuraci\u00f3n se ha alterado, cuando el examen se public\u00f3.
+seb_exam_keys_recommended=It is recommended to add exam keys for this type of configuration, because the Safe Exam Browser Configuration can \
+not be validated automatically
```
[the remark this answers is not in the document]
Comment (bgarciaentornos) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_es.properties:448]: Safe Exam Browser is not being translated elsewhere, right?

[the remark this answers is not in the document]
Comment (stetsche) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_es.properties:448]: Ahh, I see. I changed it, but it looks like it wasn't applied before. Now allright?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties (hunk 2 above)
Comment (bgarciaentornos) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties:441]:
```suggestion
seb_exam_keys_recommended=Es recomana afegir claus de prova per a aquest tipus de configuraci\u00f3, ja que la configuraci\u00f3 del Safe Exam Browser \
no pot ser validada autom\u00e0ticament
```

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_es.properties [hunk 4]
```diff
@@ -444,6 +444,8 @@ seb_exam_keys_upload_info=Para configuraciones subidas mediante fichero las clav
 Vas a tener que descargar la configuraci\u00f3n (alterada) de nuevo e insertar las claves del examen presentes en ese fichero.
 seb_exam_keys_upload_published_info=Por favor, inserta las claves del examen presentes en el fichero de configuraci\u00f3n que puedes descargar de arriba. \
 La configuraci\u00f3n se ha alterado, cuando el examen se public\u00f3.
+seb_exam_keys_recommended=Se recomienda a\u00f1adir claves de examen para este tipo de configuraci\u00f3n, ya que la configuraci\u00f3n del Safe Exam Browser \
+no se puede validar automáticamente
```
[the remark this answers is not in the document]
Comment (stetsche) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_es.properties:448]: Thanks!

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties [hunk 5]
```diff
@@ -385,6 +385,8 @@ seb_exam_keys_upload_info = Fitxategi bidez igotako konfigurazioetan, azterketar
 Deskargatu egin beharko duzu berriro konfigurazioko fitxategia (aldatuta) eta txertatu beharko dituzu horretan dauden gakoak.
 seb_exam_keys_upload_published_info = Sartu azterketaren gakoak, konfigurazioko fitxategian daudenak. Fitxategia goiko aldean deskargatu daiteke. \
 Konfigurazioa aldatu da azterketa argitaratu denean.
+seb_exam_keys_recommended=It is recommended to add exam keys for this type of configuration, because the Safe Exam Browser Configuration can \
+not be validated automatically
```
Comment (stetsche) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:389]: Hi @PabloSanRoman, could you help me with this translation please?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties (hunk 2 above)
Comment (stetsche) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_ca.properties]: Thanks!

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties (hunk 5 above)
Comment (PabloSanRoman) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:389]: Hi @stetsche , sure!. 

seb_exam_keys_recommended=Azterketara sartzeko pasahitza jartzea gomendagarria da, Safe Exam Brower-en konfigurazioa agian ez baita automatikoki balidatzen.

Comment (stetsche) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:389]: Thank you!

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Thank you very much for the great work on this.

**Pull request merged**